api/pom.xml

```diff
@@ -25,8 +25,8 @@
     </parent>
 
     <artifactId>microprofile-opentracing-api</artifactId>
-    <name>MicroProfile Opentracing API</name>
-    <description>Opentracing APIs for MicroProfile :: API</description>
+    <name>MicroProfile OpenTracing API</name>
+    <description>OpenTracing behaviors and APIs for MicroProfile :: API</description>
```

Comment:
What does "behaviors" mean for the API module?

Reply:
that should be a question for Steve :). 

I think he mean that when you use API something happens - eg. client will be traced